### PR TITLE
Show AI function messages in editable input, fix empty initial message

### PR DESCRIPTION
**Pull request by dqbd**

### CODE REVIEW

File: examples/widgets/server.py [hunk 1]
```diff
@@ -43,9 +47,9 @@ class ChatHistory(BaseModel):
 class ChatHistoryMessage(BaseModel):
     chat_history: List[BaseMessage] = Field(
         ...,
-        extra={"widget": {"type": "chat", "input": "question"}},
+        extra={"widget": {"type": "chat", "input": "question", "output": "output"}},
```
Comment (eyurtsev) [inline examples/widgets/server.py:50]: input should be location?

File: examples/widgets/server.py [hunk 2]
```diff
@@ -61,9 +65,21 @@ def chat_with_bot(input: Dict[str, Any]) -> Dict[str, Any]:
...
+def chat_message_bot(input: Dict[str, Any]) -> List[BaseMessage]:
     """Bot that repeats the question twice."""
-    return ChatMessage(content=f"Hello: {input['question']}", role="example")
+    return [
+        AIMessage(
+            content="",
+            additional_kwargs={
+                "function_call": {
+                    "name": "get_weather",
+                    "arguments": f'{{"location": "{input["location"]}"}}',
```
Comment (eyurtsev) [inline examples/widgets/server.py:76]: ```json.dumps({"location": input["location"]})```

Would be more obvious for folks I think

File: langserve/playground/src/components/ChatMessagesControlRenderer.tsx [hunk 3]
```diff
@@ -101,6 +104,15 @@ function constructMessage(
   return null;
 }
 
+function isOpenAiFunctionCall(
+  x: unknown
+): x is { name: string; arguments: string } {
+  if (typeof x !== "object" || x == null) return false;
+  if (!("name" in x) || typeof x.name !== "string") return false;
+  if (!("arguments" in x) || typeof x.arguments !== "string") return false;
```
Comment (eyurtsev) [inline langserve/playground/src/components/ChatMessagesControlRenderer.tsx:112]: Do we know behavior for an argumentless function?  empty dict encoded as json?

**Review by eyurtsev — APPROVED** [review]
Looks good to me -- let's fix the input key in the widget and merge?

File: langserve/playground/src/components/ChatMessagesControlRenderer.tsx (hunk 3 above)
Comment (dqbd) [reply langserve/playground/src/components/ChatMessagesControlRenderer.tsx:112]: Based on testing it should be an empty dict

**Pull request merged**